src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public Color getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```

Reply:
Let's see task description

In the main() method we need to create an array of figures (the size of the array can be 3 or 6, it doesn't matter). The first half of figures in this array should be generated with random parameters.

For this purpose create two more classes:

`ColorSupplier with public String getRandomColor() method - for generating random color,`

Creating enum helps you to provide different colors, but method String getRandomColor() should return String representation of Color (from enum)